astropy/coordinates/tests/test_celestial_transformations.py

```diff
@@ -176,6 +176,64 @@ def test_supergalactic():
     assert supergalactic.separation(icrs) < 0.005 * u.degree
 
 
+class TestHCRS():
+    """
+    Check HCRS<->ICRS coordinate conversions.
+
+    Uses ICRS Solar positions from get_body_barycentric.
```

Comment:
Can you clarify what this means?  It doesn't seem to use `get_body_barycentric` anywhere... I assume you mean that's how you got the values in the first place?  If so, maybe include just a quick snippet of that.